variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "origin_bucket_name" {}
```

Comment:
Do you think it makes sense to drop the `origin_` prefix here? Seems like that might already be communicated by the module name.